databrowser/src/pages/datasets/RawViewPage.vue

```diff
@@ -10,6 +10,12 @@
         <div v-if="apiResult.isFetching">Loading</div>
         <div v-else>
           <vue-json-pretty :data="apiResult.data?.data" :deep="3" show-length />
+          <div class="fixed bottom-0 left-0 py-5 w-full bg-white">
```

Comment:
Can we make this also as a component so that we can reuse it also in the detail and table view?